USFMToolsSharp/HtmlRenderer.cs

```diff
@@ -286,6 +282,10 @@ private string RenderMarker(Marker input)
                     }
                     output.AppendLine("</span>");
                     break;
+                case WMarker wMarker:
+                    output.AppendLine($"<span class=\"word-entry\"> {wMarker.Term} </span>");

```

Comment:
I think we don't want to put spaces around the word. I think this will end up with additional whitespace

Reply:
Yeah right because it'll be embedded into the verse.